src/winforms/toga_winforms/fonts.py

```diff
@@ -17,20 +18,35 @@ def points_to_pixels(points, dpi):
 
 class Font:
     def __init__(self, interface):
+        self._pfc = None  # this needs to be a class variable, otherwise we might get Winforms exceptions later
```

Comment:
I'm confused... on what condition can we get a Winforms exception? AFAICT, `self._pfc` is only referenced between lines 32 and 36... and it's recreated every time. How does this error manifest?

And, if it *does* needs to be a class variable then it should be declared on the class. This is currently declared as an instance variable.

Reply:
See the discussion between me and SyntheticBee on Discord and here: https://stackoverflow.com/questions/11829344/parameter-is-not-valid-when-draw-text-in-label-with-custom-font

The exception seems to happen, when there are too many custom fonts being used.
I'm not sure myself, where exactly the exception occurs and why the use of this instance variable (you're right: it is not a class variable) helps.